src/app/shared/NotePending.svelte

```diff
@@ -56,8 +58,11 @@
 
   $: isPending = pendings > 0
   $: isCompleted = total === success + failed + timeout
+  $: isMobile = innerWidth < 720
```

Comment:
util/html has an `isMobile` variable that should work (no real need to be dynamically responsive)